packages/core/admin/admin/src/content-manager/pages/ListView/ListViewPage.tsx

```diff
@@ -266,6 +266,7 @@ const ListViewPage = () => {
 
   return (
     <Main>
+      <Helmet title={`${contentTypeTitle} | Strapi`} />
```

Comment:
Is this in the designs? I'm just thinking its a bit inconsistent with other
places in the app

Reply:
At the moment is just says "Content Manager" other places say "Settings | Api Tokens", the whole app is inconsistent 😢